proxy/reverse.go

```diff
@@ -55,6 +57,19 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
 	proxyreq := new(http.Request)
 	*proxyreq = *clientreq
 
+	var proxybody []byte
```

Comment:
>  I could re-use?

No. 

But then we can move the body reset out of the for loop, not a huge deal though.